protected/humhub/modules/comment/models/Comment.php

```diff
@@ -151,6 +151,28 @@ public function afterSave($insert, $changedAttributes)
                 }, $mentionedUsers)]);
             }
 
+            if ($this::isSubComment($this)) {
+                $parent = $this->object_model::findOne(['id', $this->object_id]);
+                $parentUser = User::findOne(['id' => $parent->created_by]);
+
+                /**
+                 * Send notifications to only these followers:
+                 * author of the parent comment
+                 * author of the commented record
+                 */
+                $followersShouldReceiveNotification = [
```

Comment:
Shouldn't we add also all users already created a sub comment to the same parent?

Reply:
Done!